### PR TITLE
[memcached] bump version to 1.6.28; add external_ip option.

### PR DESCRIPTION
**Pull request by galkindmitrii**
- 1.6.21 is 1-year old by now.
- external_ip needed for global designate/keystone

### CODE REVIEW

File: common/memcached/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: memcached
-version: 0.3.2
+version: 0.3.3
```
Comment (businessbean) [inline common/memcached/Chart.yaml:3]: please bump it to `0.4.0` because of the new feature.

**Review by businessbean — CHANGES_REQUESTED** [review]
Please add the `external_ip` option (empty value) to the `values.yaml` plus a small comment snippet.

**Comment by galkindmitrii** [comment]
> Please add the `external_ip` option (empty value) to the `values.yaml` plus a small comment snippet.

That's already there. Or did you have something different in mind?

**Comment by businessbean** [comment]
> > Please add the `external_ip` option (empty value) to the `values.yaml` plus a small comment snippet.
> 
> That's already there. Or did you have something different in mind?

True https://github.com/sapcc/helm-charts/blob/ce4382d01b6440596812f71b4795c2069b08d5ea/common/memcached/values.yaml#L21-L22

**Review by businessbean — APPROVED** [review]
a local `helm template` looks good...

**Pull request merged**